apps/openmw/mwmechanics/character.cpp

```diff
@@ -697,7 +697,12 @@ CharacterController::CharacterController(const MWWorld::Ptr &ptr, MWRender::Anim
             mAnimation->showCarriedLeft(updateCarriedLeftVisible(mWeaponType));
         }
 
-        if(!cls.getCreatureStats(mPtr).isDead())
+        if(!cls.getCreatureStats(mPtr).isDead() ||
+		(cls.isNpc() &&
+		 mPtr.get<ESM::NPC>()->mBase->mFlags & 8)) // persistent !
```

Comment:
If you have found a new flag, the flag should be added to the enum in ESM::Npc, to esmtool and OpenCS. I kinda doubt this is correct, though. We already have ```bool mPersistent``` in ESM::Npc set based on the record flags. And what does persist have to do with mannequins?

> From http://wiki.theassimilationlab.com/mmw/Morrowind_Script_Library#Mannequins
> mannequins are npc with the persist bit, they don't play the
> death animation and just stay here

I would guess they don't play death animations because of the skipAnim call, the Persist bit should have nothing to do with it.